nipoppy/trackers/run_tracker.py

```diff
@@ -41,6 +41,7 @@
     "fmriprep": ["anat"],
     "mriqc": ["anat"],
     "tractoflow": ["anat", "dwi"],
+    "maget_brain": ["anat"]
```

Comment:
So we have a tracker for `maget_brain` but it is not tracked by `git`?

We also don't have a runscript for it... because it's not containerized?

Reply:
I guess we can ignore this now as we are switching to CLI soon?